### ISSUE TITLE
Any Ideas How to Fetch Followers or Likers?

### ISSUE THREAD

**Issue by ivanchalov** [instabot-py/instabot.py#1729]
## What is this about?
This is about a feature where Bot would be able to follow followers of specified accounts or likers of their posts. 
I see requests for this feature in telegram and few of the issue threads.

## Proposition
Can anyone help me fetch those followers/likers?
I kinda have time to dig into it and code, but need some help. Instagram does not let you get followers straight away - request for: 
```
https://www.instagram.com/username/followers/
```
Would return you to the user page. Only when you click the **button** in browser, it would open followers list. Can anyone help/explain how to emulate this through the code?..
Showing a good study material on the topic would work as well (%

**Comment by gonjumixproject** [comment]
Hi,
I am using " instaloader" to get the Follower and Followee list. I am using another python script to do that, unfortunately I could not use instaloader and instabot.py together.
Link is below if you interested;
"https://instaloader.github.io/as-module.html#instaloader.Profile.get_followers"
I currently have a bash script to;
- Run my .py code for instaloader. Get follower and followee list. Compare this lists and find the unwanted users from following list. 

I will add this "unwanted" list into the "unfollow_whitelist" section for instabot.py, but I am not really sure whether the  "unwanted" users inside of the "unfollow_whitelist" will be blocked or just unfollowed. I will test that first.

Long story short, it would be really great to have the functionality you mentioned in instabot.py..

**Comment by DarkSuniuM** [comment]
Closed all the issues till 2019-01-09, We updated the repo, Please create a new issue if you need! Thanks
